src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -168,6 +188,16 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
 
                     app.UseRequestContextAfterAuthentication<IDicomRequestContext>();
 
+                    app.UseSwagger();
+
+                    app.UseSwaggerUI(options =>
+                    {
+                        foreach (ApiVersionDescription description in provider.ApiVersionDescriptions)
+                        {
+                            options.SwaggerEndpoint($"/swagger/{description.GroupName}/swagger.yaml", description.GroupName.ToUpperInvariant());
```

Comment:
Why is this the path it supports btw? Can we simply change where these files are output because this is presumably a static file that is served?

Reply:
updated path